include/grl/ros/KukaLBRiiwaROSPlugin.hpp

```diff
@@ -330,13 +330,15 @@ namespace grl {
              break;
            case grl::flatbuffer::ArmState_StartArm:
              if (debug) {
-              ROS_INFO("Sending start!");
+               ROS_INFO("Sending StartArm!");
              }
              break;
            case grl::flatbuffer::ArmState_ShutdownArm:
              break;
            default:
-             ROS_INFO("KukaLBRiiwaROSPlugin in unsupported mode!");
+             if(debug) {
```

Comment:
maybe I shouldn't do this now? or maybe add a modulo counter so it prints only occasionally?